phpunit/class-rest-shortcodes-controller-test.php

```diff
@@ -186,10 +150,8 @@ public function data_get_item_with_field_combinations() {
 					'postId'    => self::$post_id,
 				),
 				array(
-					'html'  => '\xe2\x82\xa1',
-					'type'  => 'html',
-					'style' => '',
-					'js'    => '',
+					'html' => '<p>\xe2\x82\xa1</p>',
```

Comment:
As with the plain-english version, if the text sent in matches the text output with autop applied (I tried removing the filter from the_content, damn thing is like the flu it wouldn't go away), then should that be an error-case?